cmd/node/main.go

```diff
@@ -698,6 +706,7 @@ func processDestinationShardAsObserver(settingsConfig config.GeneralSettingsConf
 // CreateElasticIndexer creates a new elasticIndexer where the server listens on the url,
 // authentication for the server is using the username and password
 func CreateElasticIndexer(
+	ctx *cli.Context,
```

Comment:
There is no need for the entire context here. Couldn't you just pass the bool indexTransactions?

Reply:
+1

Reply:
@raduchis, @SebastianMarian - I can do that. The reasoning was that if the `NewElasticIndexer` will have more flag options, we will end up removing the bool and passing the context so we will reduce the number of parameters. Do you still think it's better to have only one bool until then?